docs/deprecated.md

```diff
@@ -50,6 +50,7 @@ The table below provides an overview of the current status of deprecated feature
 
 Status     | Feature                                                                                                                            | Deprecated | Remove
 -----------|------------------------------------------------------------------------------------------------------------------------------------|------------|------------
+Deprecated | [CLI plugins support](#cli-plugins-support)                                                                                        | v20.03.0   | -
```

Comment:
Sorry for disturbing you.

Does the deprecated `CLI plugins support` mean not to open this function to users (it is not recommended that users develop their own plugins), or does it mean that its function will be completely removed? For example `docker buildx`?

Reply:
It won't be removed (at least not in the short term) so that (e.g.) buildx can continue to work.